arm-dependencies.sls

```diff
@@ -0,0 +1,78 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+/home/servo/bin:
+  file.directory:
+    - user: servo
+    - group: servo
+    - dir_mode: 755
+    - file_mode: 644
+    
+arm64_links:
+  cmd.wait:
```

Comment:
Also, these states should require the `/home/servo/bin` state. They also shouldn't need to require the arm-dependencies state at all (you can symlink to files that don't exist yet), although this should be double-checked.